README.md

```diff
@@ -14,8 +14,8 @@ The Basics™ Station protocol enables the LoRa gateways with a reliable and sec
 
 ### Hardware
 
-* Raspberry Pi 3/4 or [balenaFin](https://www.balena.io/fin/)
-* SD card in case of the RPi 4
+* Raspberry Pi 0, 3/4 or [balenaFin](https://www.balena.io/fin/)
```

Comment:
Due to discussion at https://twitter.com/U8dEngineer/status/1484876878926209032 maybe specifically state *which* Pi Zeros are supported?